### PR TITLE
[pointwatch] support the display of true ML under level sync

### PR DESCRIPTION
**Pull request by posimagi**
For real this time.

### CODE REVIEW

**Comment by RubenatorX** [comment]
Let's not completely remove support for displaying your level synced master level.

**Comment by posimagi** [comment]
Updated.

**Comment by posimagi** [comment]
5 commits above were just to fix commit messages. 6th above actually changed code.

File: addons/pointwatch/statics.lua [hunk 1]
```diff
@@ -31,7 +31,8 @@ default_settings = {
...
         abyssea = "string.format('%d/%dXP %sMerits XP/hr:%.1fk %sJP CP/hr:%.1fk %d/%dEP Amber:%d Azure:%d Ruby:%d Pearl:%d Ebon:%d Silver:%d Gold:%d Time-Remaining:%d',xp.current,xp.tnl,max_color('%5.2f':format(math.floor(lp.current/lp.tnm*100)/100+lp.number_of_merits),lp.current/lp.tnm+lp.number_of_merits,lp.maximum_merits,58,147,191),math.floor(xp.rate/100)/10,max_color('%6.2f':format(math.floor(cp.current/cp.tnjp*100)/100+cp.number_of_job_points),cp.current/cp.tnjp+cp.number_of_job_points,500,58,147,191),math.floor(cp.rate/100)/10,ep.current,ep.tnml,abyssea.amber or 0,abyssea.azure or 0,abyssea.ruby or 0,abyssea.pearlescent or 0,abyssea.ebon or 0,abyssea.silvery or 0,abyssea.golden or 0,abyssea.time_remaining or 0)",
-        },
+        ep = "string.format('ML%d %d/%dEP EP/hr:%.1fk',ep.true_master_level,ep.current,ep.tnml,math.floor(ep.rate/100)/10)",
```
Comment (RubenatorX) [inline addons/pointwatch/statics.lua]: Why is this here?

File: addons/pointwatch/pointwatch.lua [hunk 2]
```diff
@@ -121,6 +121,7 @@ packet_handlers = {
         ep.current = p['Current Exemplar Points']
         ep.tnml = p['Required Exemplar Points']
         ep.master_level = p['Master Level']
+        ep.true_master_level = tnml[p['Required Exemplar Points']]
```
Comment (RubenatorX) [inline addons/pointwatch/pointwatch.lua]: Let's swap these. Have the true level be `master_level`, and the other be `synced_master_level`.

**Review by RubenatorX — COMMENTED** [review]
couple things

File: addons/pointwatch/statics.lua (hunk 1 above)
Comment (posimagi) [reply addons/pointwatch/statics.lua]: I didn't really want to edit the existing ones but seeing that they all include EP already I did anyway in the latest patchset.

File: addons/pointwatch/pointwatch.lua (hunk 2 above)
Comment (posimagi) [reply addons/pointwatch/pointwatch.lua]: Done

File: addons/pointwatch/pointwatch.lua [hunk 3]
```diff
@@ -120,7 +120,8 @@ packet_handlers = {
         accolades.current = p['Unity Points']
         ep.current = p['Current Exemplar Points']
         ep.tnml = p['Required Exemplar Points']
-        ep.master_level = p['Master Level']
+        ep.master_level = tnml[p['Required Exemplar Points']]
```
Comment (RubenatorX) [inline addons/pointwatch/pointwatch.lua:123]: Have you tested what happens when you've not unlocked Master Levels? Isn't this inner value `0` in that case? and if so, I'm pretty sure this would evaluate to `nil`, which I believe would maybe/probably cause string format error(s).

Comment (posimagi) [reply addons/pointwatch/pointwatch.lua:123]: I can try it on my mule, though I have no way to test the case where a character is 99 but doesn't have master levels unlocked, if that matters.

Comment (RubenatorX) [reply addons/pointwatch/pointwatch.lua:123]: afaik it should have the same behavior

Comment (posimagi) [reply addons/pointwatch/pointwatch.lua:123]: Confirmed that on a character without ML unlocked it shows "ML0" as expected.

Comment (RubenatorX) [reply addons/pointwatch/pointwatch.lua:123]: I don't see how that's possible since there is no `0` key in your table... so... how is that possible? 🤔

Comment (RubenatorX) [reply addons/pointwatch/pointwatch.lua:123]: Oh, or are you saying that even without master breaker it says that Required Exemplar points is 2500?

Comment (RubenatorX) [reply addons/pointwatch/pointwatch.lua:123]: Oh, jk, it does have 2500 in it, so no problem at all (although, in a perfect world, would be nice if it would just, not display ML/EP at all if it's not applicable but, that's too much work ;p

Comment (posimagi) [reply addons/pointwatch/pointwatch.lua:123]: I'll add that functionality at some unspecified point in the future if I don't entirely rewrite the addon first lol
